src/gsy_e/models/myco_matcher/myco_external_matcher.py

```diff
@@ -163,8 +163,11 @@ def match_recommendations(self, **kwargs) -> None:
                 # TODO: rename market_id to area_id in the BidOfferMatch dataclass
                 market = self.area_markets_mapping.get(
                     f"{recommendation['market_id']}-{recommendation['time_slot']}")
-                market.match_recommendations([recommendation])
-                recommendation["status"] = "success"
+                were_trades_performed = market.match_recommendations([recommendation])
+                if were_trades_performed:
+                    recommendation["status"] = "success"
+                else:
+                    recommendation["status"] = "Fail"
```

Comment:
I guess that you added this to be handy for the myco_sdk user, right? But If no matches were performed, is this really a `Fail` ? 

In any case I have a NIT: Please use either (`Success` and `Fail`) or (`success` and `fail`). Thanks!

Reply:
@hannesdiedrich this was requested by Andrea so I thought I could include it here, I'll still be working in this territory in my other task so I can adapt to your nit comment there if you like

Reply:
ok, please do not forget. thanks!